services/s3control/src/it/java/software.amazon.awssdk.services.s3control/S3ControlIntegrationTest.java

```diff
@@ -102,16 +102,6 @@ public void getPublicAccessBlock_NoSuchAccount() {
         }
     }
 
-    @Test
-    public void getPublicAccessBlock_NoSuchConfig() {
```

Comment:
I'm a bit concerned that we're diminishing our ability to detect issues with S3Control error model, since we currently have that issue of InvalidRequest having the wrong XML format. This is one of the few tests that verify we can parse the tests from the backend. On the other hand, we should follow the model. Can you think of another test that would be less flaky to replace this one? Too many tags maybe?

Reply:
Good call. I think we should mock the service response to verify the exception unmarshalling instead of calling the service. 

Can we merge this first since this might block the release? I'll create a backlog item to add mock tests for s3 control